script/Arcueid/Day 10/10_00_ARC10_1.txt

```diff
@@ -67,7 +67,7 @@ Opening my eyes, I greet the brisk morning.
 [sha:5e59fab2a1100f0f92f166b2d6335970609056c4]{
 -- Page 5, Offset 24272.
 -- 「あ。やっと起きたな、こいつめ」
-"Ah. You're finally awake, you little punk."
+"Ah. You're finally awake, you little rascal."
```

Comment:
"Little rascals" feels like a culturally used term to me, which is I think why I avoided it before.

Reply:
"you little punk" is also much cuter and closer in usage to the source.

Reply:
I prefer keeping it as 'punk', but if we were to change it, I'd go for just 'jerk'

Reply:
I suppose it would be neat to have some consistency in the usage of こいつめ, but on the other hand it shows up a paltry two times, so it doesn't particularly bother me.